src/pages/faq.js

```diff
@@ -39,24 +39,7 @@ const FaqHero = () => {
               animate={inView ? 'show' : 'hide'}
               initial="hide"
             >
-              <motion.h1
-                variants={basicStaggerChild}
-                className="text-uppercase knockout-bold"
-              >
-                Frequently
-              </motion.h1>
-              <motion.h2
-                variants={basicStaggerChild}
-                className="text-uppercase knockout"
-              >
-                Asked
-              </motion.h2>
-              <motion.h1
-                variants={basicStaggerChild}
-                className="text-uppercase knockout-bold"
-              >
-                Questions
-              </motion.h1>
+              <h1><span>Frequently</span> <span>Asked</span> <span>Questions</span></h1>
```

Comment:
the HTML semantics are good here, but i think James has tweaked these animations to his liking, so we should keep them.